lib/plugins/ar2.js

```diff
@@ -142,6 +142,69 @@ function init() {
     return result;
   };
 
+  ar2.updateVisualisation = function updateVisualisation(sbx) {
+    var forecast = sbx.properties.ar2 && sbx.properties.ar2.forecast.predicted;
+    if (forecast) {
+      sbx.pluginBase.addForecastPoints(ar2.forecastCone(forecast, false, sbx));
+    }
+  };
+
+  ar2.forecastCone = function forecastCone (forecast, usingRaw, sbx) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [This function's cyclomatic complexity is too high. (9)](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/774384080/issues/source?bid=2452379&fileBranchId=2479882#l152)